how-to-use-azureml/automated-machine-learning/dataprep/auto-ml-dataprep.yml

```diff
@@ -0,0 +1,8 @@
+name: auto-ml-dataprep
```

Comment:
This file is not in AzureMlCli and should not be pushed to GitHub.
It is generated as part of running the associated notebook.